### PR TITLE
Specify proxy version through flags to enable version skew tests against control plane

### PR DESCRIPTION
**Pull request by chxchx**

### CODE REVIEW

File: tests/e2e/framework/rawVM.go [hunk 1]
```diff
@@ -42,7 +42,7 @@ var (
 	clusterName   = flag.String("cluster_name", "", "The name of the istio cluster that the VM extends")
 	image         = flag.String("image", "debian-9-stretch-v20170816", "Image Name")
 	imageProject  = flag.String("image_project", "debian-cloud", "Image Project")
-	proxyTag      = flag.String("proxy_tag", "", "proxy tag that identifies a debian pkg")
+	sidecarTag    = flag.String("sidecar_tag", "", "proxy tag that identifies a debian pkg")
```
Comment (yutongz) [inline tests/e2e/framework/rawVM.go]: Can you rename it to something like `sidecar_debian_tag` or something, just to make it clear.

Comment (chxchx) [reply tests/e2e/framework/rawVM.go]: Done

**Comment by ldemailly** [comment]
can you link a doc or put in the description or link an issue explaining how we're going to test upgrades etc thanks to this ?

**Comment by chxchx** [comment]
Once the flags are in, I will include in subsequent commit how the test works along with the skew test. Basically it runs all e2e tests with previously released proxy.

**Comment by ldemailly** [comment]
there is a fairly complicated matrix of tests, older proxy is one of the cell but not the only one,
I'd like to see the overall direction described somewhere

File: tests/e2e/framework/kubernetes.go [hunk 2]
```diff
@@ -316,7 +318,7 @@ func (k *KubeInfo) generateSidecarInjector(src, dst string) error {
 		content = updateIstioYaml("sidecar_injector", *pilotHub, *pilotTag, content)
```
Comment (chxchx) [inline tests/e2e/framework/kubernetes.go]: @sebastienvas I was not sure if the sidecar_injector should use the pilot tag or the proxy tag.

**Pull request merged**